cdap-ui/app/cdap/components/AbstractWidget/GetSchemaWidget/index.tsx

```diff
@@ -58,11 +66,12 @@ const GetSchemaWidgetView: React.FC<IGetSchemaProps> = ({ extraConfig, classes }
     </span>
   );
 
+  const className = position.toLowerCase().includes('right') ? classes.buttonWrapper : '';
```

Comment:
We should define an enum for positions. The input should only be those values. We should avoid using indexof to search or parse input props

Reply:
Done